module/.golangci.yml

```diff
@@ -2,7 +2,7 @@ linters:
   enable:
       - bodyclose
       - deadcode
-      - depguard
+    # - depguard
```

Comment:
any comments as to why this was disabled? I think we should either comment why the line was removed or remove it tonally no reason to just sit there and be confusing.

Reply:
As discussed this linter is not helpful right now